OnlyR/Properties/Resources.resx

```diff
@@ -332,4 +332,7 @@
   <data name="X_SECS" xml:space="preserve">
     <value>{0} seconds</value>
   </data>
+  <data name="SETTINGS_AUDIO_FORMAT" xml:space="preserve">
+    <value>Audio Format</value>
```

Comment:
Please could we change to "Audio format" to be consistent with the capitalisation style?

Reply:
But of course.🙂Did the change.